app/parsers/bulkrax/csv_parser.rb

```diff
@@ -382,10 +382,11 @@ def file_paths
     end
 
     # Retrieve the path where we expect to find the files
-    def path_to_files
+    def path_to_files(**args)
+      filename = args.fetch(:filename, '')
+
       @path_to_files ||= File.join(
-        zip? ? importer_unzip_path : File.dirname(import_file_path),
-        'files'
+        zip? ? importer_unzip_path : File.dirname(import_file_path), 'files', filename
```

Comment:
@alishaevn If I'm reading this correctly, `filename` is effectively an optional argument. However, if `filename` ends up being an empty string, `File.join` will add a `/` to the end of the result: 

```ruby
irb(main):003:0> File.join('hello', 'world')
=> "hello/world"
irb(main):002:0> File.join('hello', 'world', '')
=> "hello/world/"
```

1. Do you know if this is problematic? Or does a trailing `/` not change the behavior? 
2. Was this change only made for bagit importers/exporters, or is the `filename` argument also used for CSV importer/exporters?

Reply:
> 2. the change was made for the bagit parser because a user can import multiple bags in a single import. so we needed a way to determine the correct path. in doing so, I needed to update this version of the method in the csv_parser to ensure it still worked.

Why make the change in the csv parser and not the bagit parser then?

Reply:
it was in the bagit parser, but now the bagit parser is inheriting from the csv parser. plus, we needed access to the file name in the `path_to_files` method in the bagit parser. which meant that result would already have the file name appended to it, but get it appended again in the previous version of the `add_path_to_file` method, which I also updated. 

it was either do some conditionals in the `add_path_to_file` method based on the parser, or just update the `path_to_files` methods that already existed in the csv and bagit parsers. I felt the latter was cleaner and more semantic.

Reply:
Ahh, I think I get it now. Thanks!